k8s-install/kubeadm/README.md

```diff
@@ -10,6 +10,10 @@ Then use kubectl to create the manifest in this directory:
 ```
 kubectl create -f canal.yaml
 ```
+or for quick, you can using github raw url 
```

Comment:
I think we should just make this the default way of installing from these instructions.

So it would look like this:

```
Then use kubectl to create the manifest in this directory:

```bash
kubectl apply -f https://raw.githubusercontent.com/projectcalico/canal/master/k8s-install/kubeadm/canal.yaml

```